specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/jobs.json

```
@@ -0,0 +1,539 @@
+    "title": "Azure Machine Learning Workspaces",
+    "description": "These APIs allow end users to operate on Azure Machine Learning Workspace resources.",
+    "version": "2020-09-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/codeJobs/{id}": {
+      "put": {
+        "tags": [
+          "CodeJob"
+        ],
+        "description": "Creates and executes a CodeJob.",
+        "operationId": "CodeJob_CreateOrUpdate",
+        "x-ms-examples": {
+          "Create or update CodeJob": {
+            "$ref": "./examples/createOrUpdateCodeJob.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CodeJobIdParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "CodeJob definition object.",
+            "schema": {
+              "$ref": "#/definitions/CodeJobCreateResourceArmDto"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated CodeJob",
+            "schema": {
+              "$ref": "#/definitions/CodeJobResourceArmDto"
+            }
+          },
+          "201": {
+            "description": "Successfully created CodeJob",
+            "schema": {
+              "$ref": "#/definitions/CodeJobResourceArmDto"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "CodeJob"
+        ],
+        "description": "Gets a CodeJob by name/id.",
+        "operationId": "CodeJob_Get",
```

Suggestion: usually operationId is in this pattern: `PuralNoun_Verb`, could we change this to 
```suggestion
        "operationId": "CodeJobs_Get",
```